src/redturtle/prenotazioni/upgrades.py

```diff
@@ -211,3 +214,106 @@ def to_1500(context):
+        "<li>${booking_time} - Orario di inizio prenotazione.</li>"
+        "<li>${booking_time_end} - Orario di fine prenotazione.</li>"
+        "<li>${booking_code} - Ticket identificativo della prenotazione da utilizzare per chiamare il cittadino allo sportello ad attesa ultimata.</li>"
+        "<li>${booking_type} - Tipologia prenotazione.</li>"
+        "<li>${booking_print_url} - Link di riepilogo prenotazione.</li>"
+        "<li>${booking_url_with_delete_token} - Link per cancellare la prenotazione.</li>"
+        "<li>${booking_user_phone} - Numero di telefono del cittadino.</li>"
+        "<li>${booking_user_email} - Email del cittadino.</li>"
+        "<li>${booking_office_contact_phone} - Telefono ufficio, se compilato.</li>"
+        "<li>${booking_office_contact_pec} - PEC ufficio, se compilata.</li>"
+        "<li>${booking_office_contact_fax} - Fax ufficio, se compilato.</li>"
+        "<li>${booking_how_to_get_to_office} - Informazioni su come raggiungere l’ufficio, se compilate.</li>"
+        "<li>${booking_office_complete_address} - Indirizzo completo dell’ufficio, se compilato.</li>"
+        "</ul>"
+    )
+    new_fields = {
+        "notify_on_submit_subject": "Prenotazione creata correttamente per ${title}",
+        "notify_on_submit_message": "notify_on_submit_message",
+        "notify_on_confirm_subject": "Prenotazione del ${booking_date} alle ${booking_time} accettata",
+        "notify_on_confirm_message": (
+            "La prenotazione ${booking_type} per ${title} è stata confermata!"
+            "Se non hai salvato o stampato il promemoria, puoi visualizzarlo su <a href=${booking_print_url}>questo link</a>"
+            "Se desideri cancellare la prenotazione, accedi a <a href=${booking_print_url}>questo link</a>"
+        ),
+        "notify_on_move_subject": ("Modifica data di prenotazione per ${title}"),
+        "notify_on_move_message": (
+            "L'orario della sua prenotazione ${booking_type} è stata modificato."
+            "La nuova data è ${booking_date} alle ore ${booking_time}."
+            "Controlla o stampa il nuovo promemoria su  <a href=${booking_print_url}>questo link</a>."
+        ),
+        "notify_on_refuse_subject": "Prenotazione rifiutata per ${title}",
+        "notify_on_refuse_message": "La prenotazione ${booking_type} del ${booking_date} delle ore ${booking_time} è stata rifiutata.",
+    }
+
+    for brain in api.portal.get_tool("portal_catalog")(
+        portal_type="PrenotazioniFolder"
+    ):
+        object = brain.getObject()
+
+        logger.info(f"[1501-1502] Updating fields on {brain.getPath()}")
+
+        object.templates_usage = RichTextValue(
+            instructions_default, "text/html", "text/html"
+        )
+
+        for name, value in new_fields.items():
+            setattr(object, name, value)
+
+
+def to_1502_upgrade_contentrules(context):
+    from plone.contentrules.engine.interfaces import IRuleStorage
```

Comment:
imports on top of the file

Reply:
Is it plone requirement? because in python we can do that, also should do that in situations like this.

Reply:
no it's not a plone requirement, ok let it here